python/amici/petab_import.py

```diff
@@ -119,11 +120,12 @@ def get_fixed_parameters(
                                   f"initial assignment for {compartments}")
 
     species = [col for col in condition_df
-               if sbml_model.getSpecies(col) is not None]
+               if not np.issubdtype(condition_df[col].dtype, np.number)
```

Comment:
why not jut ``issubclass(condition_df[col].dtype, numbers/np.number)``?

Reply:
In some other situation `issubdtype` worked whereas `issubclass` did not. Here it seems the latter is also fine.

Reply:
both fine for me, was just asking whether the latter had not worked for some reason (numbers subtypes with built-ins and numpy versions can be picky in python ime)

Reply:
That was why :D

```
  File "/home/runner/.local/lib/python3.6/site-packages/amici/petab_import.py", line 124, in get_fixed_parameters
    species = [col for col in condition_df
  File "/home/runner/.local/lib/python3.6/site-packages/amici/petab_import.py", line 125, in <listcomp>
    if not issubclass(condition_df[col].dtype, Number)
  File "/usr/lib/python3.6/abc.py", line 223, in __subclasscheck__
    if issubclass(subclass, rcls):
TypeError: issubclass() arg 1 must be a class
Command exited with non-zero status 1
```